spec/request/flipper_request_spec.rb

```diff
@@ -3,25 +3,68 @@
 require 'rails_helper'
 
 RSpec.describe 'Flipper UI', type: :request do
-  context 'Authenticated with LOA3 admin user' do
-    let(:user) { build(:user, :loa3) }
+  def bypass_flipper_authenticity_token
+    Rails.application.routes.draw do
+      mount Flipper::UI.app(
+        Flipper.instance,
+        rack_protection: { except: :authenticity_token }
+      ) => '/flipper', constraints: Flipper::AdminUserConstraint.new
+    end
+    yield
+    Rails.application.reload_routes!
+  end
 
+  context 'with authenticated admin user' do
     before do
       sign_in_as(user)
+      Settings.flipper.admin_user_emails << user.email
+      Flipper.enable(:test_feature)
     end
 
-    it 'Displays list of features' do
-      get '/flipper/features', params: nil
-      expect(response.body).to include('flipper/features')
-      assert_response :success
+    context 'with LOA1 access' do
+      let(:user) { build(:user) }
+
+      it 'does not allow feature toggling' do
```

Comment:
This test does not pass if constraint is not present due to the constraint's LOA3 requirement